docs/trellis/master/local-development.md

```diff
@@ -36,14 +37,46 @@ Trellis installs WordPress on your first `vagrant up` with `admin` as the defaul
 
 Re-provisioning is always assumed to be a safe operation. When you make changes to your Trellis configuration, you should provision the VM again to apply the changes:
 
+
+<CodeSwitcher :languages="{cli:'Trellis CLI',manual:'Manual'}">
+<template v-slot:cli>
+
+Run the following from any directory within your project:
+
+```bash
+$ trellis provision development
+```
+
+</template>
+<template v-slot:manual>
+
+Run the following from your project's `trellis` directory:
+
 ```bash
 $ vagrant provision
 ```
 
+</template>
+</CodeSwitcher>
+
 You can also provision with specific tags to only run the relevant roles:
 
+<CodeSwitcher :languages="{cli:'Trellis CLI',manual:'Manual'}">
+<template v-slot:cli>
+
+Run the following from any directory within your project:
+
 ```bash
-$ SKIP_GALAXY=true ANSIBLE_TAGS=wordpress vagrant provision
+$ trellis provision --tags users development
```

Comment:
See https://github.com/roots/docs/pull/267#discussion_r583002438
Although I dont think we have a `trellis up` alternative for `--tags`

Reply:
I'm not sure what you're asking for here

Reply:
He's saying that `trellis provision` won't work for development so there's no direct equivalent here.

🤔 I think this should just stay as `SKIP_GALAXY=true ANSIBLE_TAGS=wordpress vagrant provision` for now unfortunately.

Reply:
Should be fixed.